### DISCUSSION TITLE
Aliencoder's Extension chain - Name Vote

### DISCUSSION THREAD

**Discussion by sugoibtc** [coinselor/twitter#15] [Tweet Ideas]
🚀 Vote for NoM's First Extension Chain Name

The eagerly awaited extension chain that will enable EVM/WASM smart contracts on NoM is almost here.

Shape the future of Zenon by choosing the name for our first extension chain:

poll:
  - Supernova
  - Neutronum
  - Starfield
  - Other - Suggest your own!

Dive in the details 👇
https://forum.zenon.org/t/extension-chains-smart-contracts-for-nom/1512

**Comment by coinselor** [comment]
Zenon Virtual Machines (ZVM) are here:

The first of its kind is on the verge of completion.

- Cross-compatiblity with EVM/WASM
- Incentives for ecosystem builders
- Out-of-the-box NoM Multichain connectivity

Be a part of history. Propose a name 👇
https://forum.zenon.org/t/extension-chains-smart-contracts-for-nom/1512

**Reply by sugoibtc** [reply]
Nice edit! However I do think it'd be cool to do a poll on twitter as well, we can do 4 options in total (25 character limit per option)

For details we can redirect readers to the thread

**Reply by coinselor** [reply]
I think if we do poll we can't do link in the same tweet, we'll need to do link in a reply to the tweet. 

For a poll, I also think we need some "final candidates" by the community. Maybe for a later tweet?
